blocks/shared-styles/scss/_results-list.scss

```diff
@@ -1,53 +1,62 @@
 .results-list-container {
```

Comment:
this file should probably be deleted since results-list was converted to themes v2